InterneTaakAfhandeling.EndToEndTest/Infrastructure/ITAPlaywrightTest.cs

```
@@ -553,14 +553,15 @@ private async Task HandleAuthenticationAsync()
             try
             {
                 // Create login helper and perform fresh login every time
-                var loginHelper = new AzureAdLoginHelper(Page, username, password, s_uniqueOtpHelper);
+                var loginHelper = new AzureAdLoginHelper(Page, username, password, s_uniqueOtpHelper ?? throw new InvalidOperationException("TOTP helper not initialized"));
                 await loginHelper.LoginAsync();
 
-                // Don't save auth state - we want fresh login every time to see the flow
             }
             catch (Exception ex)
```

first catching a general exception and then make it more specific is tricky.  What if you just remove the empty catch?